util.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# clones all extensions from extensions.json into extensions folder
+clone() {
+    mkdir -p extensions
+    cd extensions
+    cat ../extensions.json | jq -r '.extensions[].repo' | while read -r line
+    do
+        git clone $line
+    done
+}
+
+# pulls all extensions from extensions.json
+pull() {
```

Comment:
- if `cd $line` fails for one extension then all the following extensions will fail because the loop will move up one level each time

```suggestion
pull() {
    cd extensions
    ext_dir=`pwd`
    cat ../extensions.json | jq -r '.extensions[].id' | while read -r line
    do
        cd $ext_dir/$line
        git pull
    done
}

```